build/webpack/webpack.finsemble-built-in.entries.json

```diff
@@ -1,20 +1,8 @@
 {
```

Comment:
Zona's taking care of these in a separate PR. But the build wouldn't function because these now defunct components were still using react-controls. I removed them.